carbon/carbon.py

```diff
@@ -37,27 +38,47 @@ def _create_carbon_url(code, **carbon_options: str) -> str:
 
 def create_code_image(code: str, **kwargs: str) -> None:
     """Generate a beautiful Carbon code image"""
-    options = Options()
-    if not bool(kwargs.get("interactive", False)):
-        options.add_argument("--headless")
-
-    service = (
-        Service(executable_path=kwargs["driver_path"])
-        if kwargs["driver_path"]
-        else Service()
-    )
+    # options = Options()
+    # if not bool(kwargs.get("interactive", False)):
+    #     options.add_argument("--headless")
+
+    # service = (
+    #     Service(executable_path=kwargs["driver_path"])
+    #     if kwargs["driver_path"]
+    #     else Service()
+    # )
 
     destination = kwargs.get("destination", os.getcwd())
     prefs = {"download.default_directory": destination}
-    options.add_experimental_option("prefs", prefs)
+    # options.add_experimental_option("prefs", prefs)
+    
 
     if kwargs.get("disable-dev-shm", False):
         options.add_argument("disable-dev-shm-usage")
 
-    url = _create_carbon_url(code, **kwargs)
-    with webdriver.Chrome(service=service, options=options) as driver:
-        driver.get(url)
-        driver.find_element(By.ID, "export-menu").click()
-        driver.find_element(By.ID, "export-png").click()
-        # make sure it has time to download the image
+    with sync_playwright() as p:
+        browser = p.chromium.launch(headless=False)
```

Comment:
With Option going away (no more Selenium) what about making this an environment variable, I would make this configurable.